ui/src/lib/gql/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from "./queries";
 export * from "./provider/";
+export * from "./__gen__/graphql";
```

Comment:
We decided that this wasn't a good idea because of type conflicts, no? 🤔

Reply:
I need the types in the `/types`
I can use the directly `__gen__` folder :thinking: 
I am still not sure, I don't think we should export....
to export or not to export... :thinking:

Reply:
Ah maybe I left this up to you... I guess personally I feel like knowing something is generated has its merits, so making imports need to specify the __gen__ file is good. If we don't do that, making the __gen__ file for generated code is a little pointless and we should generate directly in the lib/gql dir.
Make sense? Agree disagree?

Reply:
Makes sense, not exporting it. It should stay abstracted away in the `gql` directory